config/application.yml

```
@@ -11,8 +11,8 @@ spring:
           default-label: main
           search-paths: dev, prod
           basedir: config
-server:
-  port: 8888
+#server:
```

YAMLlint (1.35.1)

[warning] missing starting space in comment

(comments)